docs/contributing/getting-started.md

```diff
@@ -29,17 +29,17 @@ make build.push IMAGE=your-registry/external-dns
 
 # Design
 
-ExternalDNS's sources of DNS records live in package [source](../../source). They implement the `Source` interface that has a single method `Endpoints` which returns the represented source's objects converted to `Endpoints`. Endpoints are just a tuple of DNS name and target where target can be an IP or another hostname.
+ExternalDNS's sources of DNS records live in package [source](https://github.com/kubernetes-sigs/external-dns/tree/master/source). They implement the `Source` interface that has a single method `Endpoints` which returns the represented source's objects converted to `Endpoints`. Endpoints are just a tuple of DNS name and target where target can be an IP or another hostname.
```

Comment:
the link should IMO link the markdown, not a deeplink:
`[source][../source.md]` should work

Reply:
It cannot, it's a link to a go source, not a markdown file. AFAIK, it cannot be displayed by mkdocs.